executor/ddl.go

```diff
@@ -152,6 +154,44 @@ func (e *DDLExec) executeCreateTable(s *ast.CreateTableStmt) error {
 	return errors.Trace(err)
 }
 
+func (e *DDLExec) executeCreateView(s *ast.CreateViewStmt) error {
+	ident := ast.Ident{Schema: s.View.Schema, Name: s.View.Name}
+	// SelectFields represents the field expression text in SelectStmt and
+	// saves it as a string to establish the mapping relationship of ColList
+	var selectFields []string
+	selectstmt := s.Select.(*ast.SelectStmt)
+	fields := selectstmt.Fields.Fields
+	for _, field := range fields {
+		if field.AsName.O == "" {
+			selectFields = append(selectFields, field.Expr.Text())
```

Comment:
The `Expr` is null when the select field is a wild card, so this will cause a panic.